rockcraft.yaml

```diff
@@ -26,5 +26,8 @@ parts:
     source: https://github.com/ory/kratos
     source-type: git
     source-tag: v0.13.0
-    stage:
-      - bin/kratos
+    override-build: |
```

Comment:
Why are we overriding the plugin build?

Reply:
The default plugin creates a binary for every command. I am not sure if this is intentional or a kratos bug, but this resulted in getting the same binary for every kratos command. For this reason we used the `stage` parameter to place only one of the binaries in the final image. But if we use the `stage` parameter then the `stage-packages` are not placed into the primed image.